docs/fastdds/api_reference/dds_pim/core/Time_t/Duration_t.rst

```diff
@@ -0,0 +1,7 @@
+.. rst-class:: api-ref
+
+eprosima::fastdds::dds::Duration_t
+----------------------------------
+
+.. doxygentypedef:: eprosima::fastdds::dds::Duration_t
+    :project: FastDDS
```

Comment:
[Not found](https://eprosima-fast-rtps--870.org.readthedocs.build/en/870/fastdds/api_reference/dds_pim/core/Time_t/Duration_t.html)